packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -0,0 +1,294 @@
+          <Text>Cras semper auctor neque vitae tempus quam pellentesque nec. Maecenas ultricies mi eget mauris pharetra et ultrices neque ornare. Vulputate enim nulla aliquet porttitor lacus luctus accumsan tortor posuere. Pellentesque habitant morbi tristique senectus et. Ipsum dolor sit amet consectetur adipiscing elit pellentesque. Sem et tortor consequat id porta nibh venenatis. Viverra nibh cras pulvinar mattis nunc sed blandit. Urna porttitor rhoncus dolor purus. Vivamus arcu felis bibendum ut. Cras sed felis eget velit aliquet. Sed tempus urna et pharetra pharetra. Viverra adipiscing at in tellus integer feugiat scelerisque varius morbi. Ac tortor vitae purus faucibus ornare suspendisse sed nisi lacus. Ultrices neque ornare aenean euismod elementum nisi quis eleifend quam. Vel turpis nunc eget lorem. Quisque egestas diam in arcu cursus euismod quis viverra. At tempor commodo ullamcorper a lacus vestibulum sed. Id aliquet lectus proin nibh nisl condimentum id venenatis. Quis viverra nibh cras pulvinar. Purus in mollis nunc sed.</Text>
+        </React.Fragment>
+      </Content>
+      <Footer>
+        <Button autoFocus variant="cta" onPress={close}>Acknowledge</Button>
+      </Footer>
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+## Events
+
+For Dialogs, user defined callbacks should be chained with the DialogTrigger's `close` function in the press/click handler
+of the Dialog's action buttons. The following example alerts if the Dialog's save or cancel is clicked.
+
+```tsx example
+function Example() {
+  let alertSave = (cb) => {
+    cb();
+    alert('Profile saved!');
+  }
+
+  let alertCancel = (cb) => {
+    cb();
+    alert('Profile not saved!');
+  }
+
+  return (
+    <DialogTrigger>
+      <ActionButton>Set Profile</ActionButton>
+      {(close) => (
+        <Dialog>
+          <Header><Heading>Profile</Heading></Header>
+          <Divider />
+          <Footer>
+            <Button autoFocus variant="cta" onPress={() => alertSave(close)}>Save</Button>
+            <Button variant="secondary" onPress={() => alertCancel(close)}>Cancel</Button>
+          </Footer>
+          <Content>
+            <Form>
+              <TextField label="Name" />
+              <Checkbox>Make private</Checkbox>
+            </Form>
+          </Content>    
+        </Dialog>
+      )}
+    </DialogTrigger>
+  );
+}
+```
+
+Additionally, DialogTrigger accepts an `onOpenChange` prop which is triggered whenever the Dialog is opened or closed. 
+For more information on the `onOpenChange` prop see [DialogTrigger page](TODO LINK HERE).
```

Comment:
After several iterations, decided that DialogTrigger is probably best left as a separate docs page. Things like the "close as a function", `type`, and other trigger specific props will be discussed there